app/actions/services/service_instance_delete.rb

```diff
@@ -87,5 +88,10 @@ def build_fetch_job(service_instance)
         {},
       )
     end
+
+    def log_audit_event(service_instance)
+      event_method = service_instance.managed_instance? ? :record_service_instance_event : :record_user_provided_service_instance_event
```

Comment:
Note: this function was moved from the service_instance_deprovisioner.rb. In this PR, the user provided service case is not tested (and it wasn't in the old code either). Test coverage for this line is added in PR #1059.